src/plugins/ui_actions_enhanced/common/types.ts

```diff
@@ -26,5 +26,5 @@ export type SerializedEvent = {
 };
 
 export type DynamicActionsState = {
-  events: SerializedEvent[];
+  events?: SerializedEvent[];
```

Comment:
We previously relied on the state container from `DynamicActionManager` to fetch the current events, and that had a default value of `events = []` - however, since I switched to using the `dynamicActionsState$` publishing subject as described [here](https://github.com/elastic/kibana/pull/178896/files#r1530525558), there is no longer a default value - so, treating is as optional is safer.